index.html

```diff
@@ -68,6 +68,10 @@ <h2 id="timer">
         { src: 'img/charlize-theron.jpg', alt: "Charlize Theron" },
         { src: 'img/lisa-simpson.jpg', alt: "Lisa Simpson" },
         { src: 'img/the-captain.jpg', alt: 'The Captain' },
+        { src: 'img/greta.jpg', alt: "Greta" },
```

Comment:
How about this for a caption? ~~Also suggesting removing the leading whitespace.~~

```suggestion
        { src: 'img/greta.jpg', alt: "Greta", fullcaption: "How dare you leave your screen unlocked!" },
```

Reply:
Added the full caption. Thanks!